test/support/integration/enterprise_server.ex

```diff
@@ -74,6 +78,10 @@ defmodule Livebook.EnterpriseServer do
     {:reply, url, %{state | url: url}}
   end
 
+  def handle_call({:call_rpc, function, args}, _from, state) do
+    {:reply, call_erpc_function(state.node, function, args), state}
+  end
```

Comment:
I don’t think you need this handle_call for the rpc. Instead do a call on the EnterpriseServer to get the node and then do the call on the test process itself. :)

In other words, try to keep the callbacks to a minimum. If you can only query the state and do the work on the client, the better! But it is not always possible.

Reply:
I think you can do the same for dropping that database. :)